lmdeploy/turbomind/turbomind.py

```diff
@@ -207,16 +205,20 @@ def _forward_callback(self, result, ctx):
         self.que.put((False, result))
 
     def _forward_thread(self, inputs):
+        instance_comm = self.tm_model.model_comm.create_instance_comm(
```

Comment:
If `create_instance_comm` being put to a local thread, will it cost more time?

Reply:
https://github.com/InternLM/lmdeploy/blob/main/src/turbomind/models/llama/LlamaInstanceComm.h#L10
https://github.com/InternLM/lmdeploy/blob/main/src/turbomind/triton_backend/llama/LlamaTritonModel.cc#L388
InstanceComm only contain a pointer and a Barrier object (which is an adapter of pthread_barrier_t). The cost should be ignorable.

Reply:
> If `create_instance_comm` being put to a local thread, will it cost more time?

I tested it this morning. Did not influence performance.